diff --git a/dist/index.node.js b/dist/index.node.js
index ba8d34a963e9314b1790f4b17e280023a55955a3..437d7a3e8da418316abba99a8308dcf04c82ef50 100644
--- a/dist/index.node.js
+++ b/dist/index.node.js
@@ -18926,9 +18926,9 @@ function assertValue(v2, errorMessage) {
 
 // src/index.node.ts
 var satori = El.default || El;
-var fontData = fs.readFileSync(fileURLToPath(join(import.meta.url, "../noto-sans-v27-latin-regular.ttf")));
-var yoga_wasm = fs.readFileSync(fileURLToPath(join(import.meta.url, "../yoga.wasm")));
-var resvg_wasm = fs.readFileSync(fileURLToPath(join(import.meta.url, "../resvg.wasm")));
+var fontData = fs.readFileSync(fileURLToPath(new URL("./noto-sans-v27-latin-regular.ttf", import.meta.url)));
+var yoga_wasm = fs.readFileSync(fileURLToPath(new URL("./yoga.wasm", import.meta.url)));
+var resvg_wasm = fs.readFileSync(fileURLToPath(new URL("./resvg.wasm", import.meta.url)));
 var initializedResvg = initWasm(resvg_wasm);
 var initializedYoga = initYoga(yoga_wasm).then((yoga2) => Ll(yoga2));
 var ImageResponse = class extends Response {
